bodo/libs/_decimal_ext.cpp

```diff
@@ -417,7 +417,8 @@ array_info* decimal_array_sign_py_entry(array_info* arr_) {
     }
 }
 
-int8_t decimal_scalar_sign_py_entry(arrow::Decimal128 val) {
+int8_t decimal_scalar_sign_py_entry(uint64_t in_low, int64_t in_high) {
```

Comment:
Looks like this PR should be on top of another one. Or just merge the other to main first.